src/main/java/org/cryptomator/ui/fxapp/FxApplicationWindows.java

```diff
@@ -65,26 +66,26 @@ public FxApplicationWindows(@PrimaryStage Stage primaryStage, //
 								QuitComponent.Builder quitWindowBuilder, //
 								UnlockComponent.Factory unlockWorkflowFactory, //
 								UpdateReminderComponent.Factory updateReminderWindowFactory, //
-								DokanySupportEndComponent.Factory dokanySupportEndWindowBuilder, //
 								LockComponent.Factory lockWorkflowFactory, //
 								ErrorComponent.Factory errorWindowFactory, //
 								VaultOptionsComponent.Factory vaultOptionsWindow, //
 								ShareVaultComponent.Factory shareVaultWindow, //
-								ExecutorService executor) {
+								ExecutorService executor, //
+								Provider<Dialogs> dialogsProvider) {
```

Comment:
Wrapping inside a `Provider` is not needed anymore, since on each method call in Dialogs we are alawys creating a new builder.

Reply:
Also adjust the other places where a Provider is used.